releng/travis-script.sh

```diff
@@ -71,6 +71,47 @@ esac
 set -o xtrace
 make print-vars
 
+case "$TRAVIS_EVENT_TYPE" in
+    cron)
+        printf "========\nRunning Envoy tests...\n"
+
+        # Delete VM if it already exists
+        gcloud beta compute instances delete envoy-tests-ambassador --zone us-east1-b --quiet || true
```

Comment:
What are our thoughts about running tests in the gcp environment from the local env?

---

I'm thinking something like:

1. Turn this into a makefile task (or a script makefile calls)
2. Make the gcp instance name be something like: `envoy-tests-ambassador-$(date +%s)`
3. Have the cleanup action list all running hosts, and check if they've been powered on for more than 24 hours. If so shutdown. (Or maybe using gcp's cloud scheduler to do it automatically?)

---

Do you think it'd be worthwhile to test this part of the gcloud script? Or too much work for not much benefit?